mslib/msui/mscolab.py

```diff
@@ -209,8 +214,15 @@ def enable_login_btn(self):
     def connect_handler(self):
         try:
             url = str(self.urlCb.currentText())
-            r = requests.get(url_join(url, 'status'))
-            if r.text == "Mscolab server":
+            auth = "mscolab", self.httpPasswordLe.text()
+            s = requests.Session()
+            s.auth = auth
+            s.headers.update({'x-test': 'true'})
+            r = s.get(url_join(url, 'status_auth'), timeout=(2, 10))
+            if r.status_code == 401:
+                self.set_status("Error", 'Server authentication were incorrect.')
+            elif r.text == "Mscolab server":
```

Comment:
we should only have a status code here, 200 would be good.

Reply:
done.